templates/python/package/services/service.py.twig

```diff
@@ -24,12 +24,35 @@ from ..enums.{{ name | caseSnake }} import {{ name | caseUcfirst }};
 
 class {{ service.name | caseUcfirst }}(Service):
 
-    def __init__(self, client):
+    def __init__(self, client) -> None:
         super({{ service.name | caseUcfirst }}, self).__init__(client)
 {% for method in service.methods %}
 
-    def {{ method.name | caseSnake }}(self{% if method.parameters.all|length > 0 %}, {% endif %}{% for parameter in method.parameters.all %}{{ parameter.name | escapeKeyword | caseSnake }}: {{ parameter | getPropertyType(method) | raw }}{% if not parameter.required %} = None{% endif %}{% if not loop.last %}, {% endif %}{% endfor %}{% if 'multipart/form-data' in method.consumes %}, on_progress = None{% endif %}):
-        """{{method.title}}"""
+    def {{ method.name | caseSnake }}(self{% if method.parameters.all|length > 0 %}, {% endif %}{% for parameter in method.parameters.all %}{{ parameter.name | escapeKeyword | caseSnake }}: {{ parameter | getPropertyType(method) | raw }}{% if not parameter.required %} = None{% endif %}{% if not loop.last %}, {% endif %}{% endfor %}{% if 'multipart/form-data' in method.consumes %}, on_progress = None{% endif %}) -> Dict[str, Any]:
+        """
+        {% autoescape false %}{{ method.description | replace({"\n": "\n        "}) }}{% endautoescape %}
+{% if method.parameters.all|length > 0 or 'multipart/form-data' in method.consumes %}
+
+        Parameters
+        ----------
+        {% for parameter in method.parameters.all %}{{ parameter.name | escapeKeyword | caseSnake }} : {{ parameter | getPropertyType(method) | raw }}
+            {% autoescape false %}{{ parameter.description | replace({"\n": "\n            "}) }}{% endautoescape %}
+
+        {% endfor %}{% if 'multipart/form-data' in method.consumes %}
+        on_progress : callable, optional
+            Optional callback for upload progress
+        {% endif %}{% endif %}
+
+        Returns
+        -------
+        dict
+            API response as a dictionary
+
```

Comment:
I dont think all the endpoints return dicts. for example getFilePreview . Can we check